tamr_unify_client/_custom_encoder.py

```diff
@@ -0,0 +1,67 @@
+"""Adaptation of the Python standard library JSONEncoder to encode `NaN` as 'null'
```

Comment:
Should we rename this file to `_ignore_nan_encoder.py`?

Reply:
Done